doc/man3/RAND_DRBG_new.pod

```diff
@@ -15,11 +17,20 @@ RAND_DRBG_free
 
  #include <openssl/rand_drbg.h>
 
+ RAND_DRBG *RAND_DRBG_new_ex(OPENSSL_CTX *ctx,
```

Comment:
RAND_DRBG_ctx_new() maybe?

Reply:
Hmm. I'm dubious about adding that as a new convention. What if I later have to extend `FOO_CTX_new()` to add an `OPENSSL_CTX` param? Calling it `FOO_CTX_ctx_new` sounds odd.